### PR TITLE
Feature/cip UI 1960 blcharts integration

### PR DESCRIPTION
**Pull request by robgannon-eaton**
It is failing the link checker test. The links do work but they are for things like figma and sharepoint, and seems to be just the checker having access issues?

#### Any specific feedback you are looking for?

Ive added this to the global index.css to remove the border in the ImageGrid Component but only for these stand alone images

```
/* Remove border from ImageGrid for BrightlayerCharts integration images*/
img[src*="foundations-chart"],
img[src*="datagranularity"],
img[src*="barcharts-component"],
img[src*="line-chart-component"],
img[src*="piechart-component"] { 
    border: none;
}
```
-

### CODE REVIEW

File: src/app/components/visualizations/BLCIconButton.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    padding-bottom: 2px;
+`;
+
+type BLCIconButtonProps = {
+    icon: SvgIconComponent;
+    url: string;
+    label: string;
+};
+
+export const BLCIconButton: React.FC<BLCIconButtonProps> = ({ icon: icon, url, label }) => {
```
[the remark this answers is not in the document]
Comment (robgannon-eaton) [reply src/app/components/visualizations/BLCIconButton.tsx:38]: changed to just icon

File: src/app/components/visualizations/BLCIconButton.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    const theme = useTheme();
+    const primaryColor = theme.palette.primary.main;
+    const navigate = useNavigate();
+    let IconComp;
+    if (icon) IconComp = icon;
```
[the remark this answers is not in the document]
Comment (robgannon-eaton) [reply src/app/components/visualizations/BLCIconButton.tsx:43]: changed to `const IconComp = icon;`

File: src/app/components/visualizations/BLCIconButton.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        if (url.startsWith('http://') || url.startsWith('https://')) {
+            window.open(url, '_blank');
+        } else if (url) {
+            navigate(url);
+        }
+    };
+    return (
+        <div style={{ display: 'inline-block' }}>
+            <StyledIconButton onClick={handleClick} bordercolor={primaryColor}>
+                {IconComp ? <IconComp fill={primaryColor} /> : ''}
```
[the remark this answers is not in the document]
Comment (robgannon-eaton) [reply src/app/components/visualizations/BLCIconButton.tsx:55]: done

File: src/app/components/visualizations/GettingStartedTabs.tsx [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+    const TabIcon1 = tabs[0].icon;
+    const TabIcon2 = tabs[1].icon;
+    const TabIcon3 = tabs[2].icon;
+    const TabIcon4 = tabs[3].icon;
+
+    const handleChange = (event: React.SyntheticEvent, newValue: number): void => {
+        setValue(newValue);
+    };
+
+    const getTabColor = (tabValue: number, tab: number): string => {
```
[the remark this answers is not in the document]
Comment (robgannon-eaton) [reply src/app/components/visualizations/GettingStartedTabs.tsx:111]: done

File: src/app/components/visualizations/GettingStartedTabs.tsx [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+                    TabIndicatorProps={{
+                        style: {
+                            display: 'none',
+                        },
+                    }}
+                    value={value}
+                    onChange={handleChange}
+                    aria-label="basic tabs example"
+                >
+                    <StyledTab
```
[the remark this answers is not in the document]
Comment (robgannon-eaton) [reply src/app/components/visualizations/GettingStartedTabs.tsx:129]: done

File: src/docs/design/intro.mdx [hunk 6]
```diff
@@ -30,60 +36,89 @@ Brightlayer UI has design guidelines that serve as the foundation for the design
 
 # General Guidance
 
-The best designs generally start with mobile first, and that's what we recommend as well when working with the Brightlayer UI design system. It's much easier to scale designs up than to scale them down. We also strongly encourage you to perform extensive user research to truly understand your users and their needs, skills, preferences, and tendencies.
+Eaton hardware (such as power meters or a UPS) generate a lot of data, and we often convert that data into actionable insights. Eaton software should make energy management problems easier to solve, and communicate data insights efficiently.
 
-## Web Applications
+Brightlayer UI is here to support designers in creating products that help our customers solve problems and add value to their energy management. These principles can help:
```
[the remark this answers is not in the document]
Comment (robgannon-eaton) [reply src/docs/design/intro.mdx:41]: I cant really comment on copy as it matches the designs - https://www.figma.com/design/kyqxbVCTeu3nfQpleNfXR8/Merge-with-BLUI-docit-website?node-id=29-122&node-type=canvas&t=tkxQ9l8vDX7r1coQ-0
Bryan Bunn ( bryannbunn@eaton.com ) is the designer on our team working on the integration, if you would like to contact him.

**Review by surajeaton — APPROVED** [review]

**Pull request merged**